package.json

```diff
@@ -50,7 +50,7 @@
     "@ledgerhq/hw-transport-http": "6.26.0",
     "@ledgerhq/hw-transport-node-hid-singleton": "6.26.0",
     "@ledgerhq/ledger-core": "6.14.5",
-    "@ledgerhq/live-common": "21.36.2",
+    "@ledgerhq/live-common": "file:.yalc/@ledgerhq/live-common",
```

Comment:
Don't do this, if you are depending on some specific pr from live common, reference it here. That way someone else can checkout the branch and build, without having to also do the yalc setup locally.

It is also healthy because it keeps things tightly coupled until merged. 

Im on my phone but it's something like this
```
    "@ledgerhq/live-common": "https://github.com/ledgerhq/live-common.git#commitHash",
```